webapp/app/components/ui/collapsible.tsx

```diff
@@ -0,0 +1,9 @@
+import * as CollapsiblePrimitive from "@radix-ui/react-collapsible";
```

Comment:
I feel that we already had something to do collapsible things.

Reply:
added by shadcn!